seccomp.go

```diff
@@ -793,6 +793,30 @@ func (f *ScmpFilter) GetLogBit() (bool, error) {
 	return true, nil
 }
 
+// GetSsbBit returns the current state the SSB bit will be set to on the filter
+// being loaded, or an error if an issue was encountered retrieving the value.
+// The SSB bit tells the kernel that a seccomp user is not interested in enabling
+// Speculative Store Bypass mitigation.
+// The Log bit is only usable when libseccomp API level 4 or higher is
+// supported.
+func (f *ScmpFilter) GetSsbBit() (bool, error) {
+	ssb, err := f.getFilterAttr(filterAttrSSB)
+	if err != nil {
+		api, apiErr := getAPI()
+		if (apiErr != nil && api == 0) || (apiErr == nil && api < 4) {
+			return false, fmt.Errorf("getting the log bit is only supported in libseccomp 2.5.0 and newer with API level 4 or higher")
```

Comment:
You don't mean to say "log bit" here do you?  You mean "SSB flag", right?

Reply:
I'm sorry. I fixed it.